### DISCUSSION TITLE
x12s_Touch_MOD

### DISCUSSION THREAD

**Discussion by rcman45** [EdgeTX/edgetx#2081] [Q&A]
j'ai montè un écran tactile sur ma FrSky X12S mais il fonctionne a l’envers
que faut il modifier pour le bon fonctionnement ?

**Comment by pfeerick** [comment]
Interesting... same has been mentioned here: https://github.com/EdgeTX/edgetx/issues/2063

**Comment by rcman45** [comment]
en utilisant le firmware https://github.com/wimalopaan/Electronics/blob/main/edgetx/firmware/x12s/etx_x12s_en_2.8.0_wm_touch_3.bin 
cela fonctionne impeccable, mais pas une construction personel ou le firmware etx_x12s_de_2.8.0_wm_touch_4.bin et etx_x12s_de_2.8.0_wm_touch_5.bin de wimalopaan qui fonctionne toujour a l'envers

je tiens a signaler que le capteur n'est pas inversé car j'ai acheté un ecran complet

**Comment by rcman45** [comment]
J'ai fait une erreur
sur les version de Wimalopaan
Ce sont les versions 1,2,3 et 4 qui fonctionne bien (deu ou eng)
les 5 et 6 sont a l'envers
aussi les construction personnel a l'envers
